clean_records.sh

```diff
@@ -0,0 +1,2 @@
+LAST_MONTH=$(date --date='-1 month' +'%Y-%m')
+rm "tmp/$LAST_MONTH-"*
```

Comment:
maybe we could move this script inside the downloads script as a flag instead since the user would probably not always use the `tmp` directory.

Reply:
I would use a separate script because using crontab a user can in parallel run this only script to delete the old files

Reply:
we can achieve the same thing using a flag with the download script, but i think your approach is better since it cannot be accidental when you literally run "clean_records". But I think you still need to be able to delete stuff in a different folder and thus need to accept an argument for a directory

Reply:
done